lib/assignment/widgets/assignment_detail_screen.dart

```diff
@@ -25,14 +24,12 @@ class _AssignmentDetailScreenState extends State<AssignmentDetailScreen>
   Widget build(BuildContext context) {
     final s = context.s;
 
-    return CachedRawBuilder<Assignment>(
+    return FancyCachedBuilder<Assignment>.handleLoading(
       controller: widget.assignmentId.controller,
-      builder: (context, update) {
-        final assignment = update.data;
-        return CachedRawBuilder<User>(
+      builder: (context, assignment, isFetching) {
+        return FancyCachedBuilder<User>.handleLoading(
           controller: services.storage.userId.controller,
-          builder: (context, update) {
-            final user = update.data;
+          builder: (context, user, isFetching) {
             final showSubmissionTab =
```

Comment:
Now that loading is handled for us, we should be able to get rid of null-awareness when accessing the user